mappings/net/minecraft/world/gen/config/feature/BlockClusterFeatureConfig.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS btc net/minecraft/world/gen/config/feature/BlockClusterFeatureConfig
```

Comment:
Im not sure how we are doing it in other places, but FeatureConfigBlockCluster makes more sense to me? What do other think? This would need to be changed on the other classes as well.

Reply:
We haven't set on a standard. Leave it be.